cmd/lakectl/cmd/log.go

```diff
@@ -37,6 +39,7 @@ var logCmd = &cobra.Command{
 		amount := MustInt(cmd.Flags().GetInt("amount"))
 		after := MustString(cmd.Flags().GetString("after"))
 		limit := MustBool(cmd.Flags().GetBool("limit"))
+		dot := MustBool(cmd.Flags().GetBool("dot"))
```

Comment:
Will the combination of `dot` with `after`/`objectsList`/`prefixesList` make sense?

Reply:
I believe it will - it will show a partial graph which is what the user requested (and is not the default behavior)